bongo/locallib.php

```diff
@@ -37,12 +37,11 @@
  *
  */
 
-require_once(__DIR__ . '/../../../config.php');
```

Comment:
All references to config.php are not necessary unless the page is user-visible. As such, this one has been removed.